docker-stack-wait.sh

```diff
@@ -84,6 +87,12 @@ while [ "$stack_done" != "1" ]; do
   # run get_service_ids outside of the for loop to catch errors
   service_ids=$(get_service_ids)
   for service_id in ${service_ids}; do
+    if [ "$opt_v" = "1" ]; then
+      # The idea of showing logs is to make it easier to debug startup issues,
+      # so we'll include a bit more then necessary to catch enough context to make debugging easier.
+      docker service logs --since 1m -f "$service_id" &
```

Comment:
Putting this with a `-f` in the background inside of the while loop could result in lots of duplicate logs, one for each cycle of the loop while waiting for services to go healthy. I think it makes more sense to either launch it once at the start, or preferably in the foreground (without a `-f`) at the end after an error is encountered.

Reply:
You're right, in my testing I only tested with fast starting containers, so I didn't notice that. I'll try to move it to the end.